tests/countries/test_turks_and_caicos_islands.py

```
@@ -0,0 +1,156 @@
+                
+    def test_sovereign_birthday(self):
+        holidays_2022 = TurksAndCaicosIslands(years=2022)
+        self.assertIn(date(2022, 6, 13), holidays_2022)
+        self.assertEqual(holidays_2022[date(2022, 6, 13)], "Queen's Birthday")
+        
+        holidays_2023 = TurksAndCaicosIslands(years=2023)
+        self.assertIn(date(2023, 6, 12), holidays_2023)
+        self.assertEqual(holidays_2023[date(2023, 6, 12)], "King's Birthday")
+                
+    def test_national_youth_day(self):
+        self.assertHolidayName("National Youth Day", (f"{year}-09-{day}" for year, day in 
```

Ruff (0.8.2)

Trailing whitespace

Remove trailing whitespace

(W291)